app/controllers/tournaments_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TournamentsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def saveTournamentResults
```

Comment:
Always use snake_case - same goes for all of the other endpoints
```suggestion
  def save_tournament_results
```

Reply:
how its can be? 
how you can config a route with snake_case? there is a way to configure route as usual and route it to the controller with snake_case?